blogposts/2021/optimizing-a-code-intelligence-commit-graph-1.md

```diff
@@ -2,23 +2,29 @@
 title: "Optimizing a code intelligence commit graph (Part 1)"
 author: Eric Fritz
 authorUrl: https://eric-fritz.com
+description: "Read about our journey to enabling Sourcegraph to respond to requests on commits missing an index quickly, with precise results."
 publishDate: 2021-05-24T18:00-07:00
 tags: [blog]
 slug: optimizing-a-code-intel-commit-graph
-heroImage: https://user-images.githubusercontent.com/82806852/118469885-10848200-b6fe-11eb-8670-43300ef355a8.jpg
+heroImage: /blog/optimizing-code-intelligence-commit-graph.png
+socialImage: /blog/optimizing-code-intelligence-commit-graph.png
```

Comment:
@efritz Thanks for letting me know. I thought this was a small enough image that it'd be ok (going by https://about.sourcegraph.com/handbook/marketing/creating_blog_posts#adding-images) but if I'm honest it's mainly because I haven't yet tackled how to upload to the bucket 🙈

Reply:
Yeah small images are fine. I think this is something we should add some additional dev tooling around (have a marketing cli tool that uploads images from your machine, other common tasks). I'm just mentioning it because it was easy to flag here, no action necessary on this one :)